email_mailbox/src/utils/electronEventInterface.js

```diff
@@ -118,14 +121,40 @@ const stopGettingEvents = () => {
   processPendingEvents({});
 };
 
-const parseAndStoreEventsBatch = async ({ events, hasMoreEvents }) => {
+const getAccountInfo = async (recipientId, domain) => {
+  if (!recipientId || !domain) {
+    return {};
+  }
+  const accountRecipientId =
+    domain === appDomain ? recipientId : `${recipientId}@${domain}`;
+  const [account] = await getAccountByParams({
```

Comment:
Wouldn't be better if you get the desired account from `myAccount` instead of a db-call?